src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -19,7 +19,7 @@ public class DurableTaskOptions
         public HttpOptions HttpSettings { get; set; }
 
         /// <summary>
-        /// Gets or sets default task hub name to be used by all <see cref="DurableOrchestrationClient"/>,
+        /// Gets or sets default task hub name to be used by all <see cref="DurableClient"/>,
```

Comment:
Similar comment here about referencing the public facing interfaces instead of internal concrete type.